kimchi/src/circuits/lookup/lookups.rs

```diff
@@ -296,7 +296,7 @@ impl LookupPattern {
         match self {
             LookupPattern::Xor | LookupPattern::ChaChaFinal | LookupPattern::RangeCheck => 4,
             LookupPattern::Lookup => 3,
-            LookupPattern::ForeignFieldMul => 2,
+            LookupPattern::ForeignFieldMul => 1,
```

Comment:
Perhaps we want to call it something more meaningful like `AtMost3Bits` or similar

Reply:
I'm not sure.  You may be right that it's a good idea, but let's keep the idea open for a subsequent PR.